.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] The PR follows our guidelines: 
+
+    - [Collaborator-Guide](https://github.com/expressjs/express/blob/master/Collaborator-Guide.md)
+    - [Contributing](https://github.com/expressjs/express/blob/master/Contributing.md)  
+    - [Release-Process](https://github.com/expressjs/express/blob/master/Release-Process.md)
+    
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+## PR Type
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
+[ ] Feature
+[ ] Code style update (formatting, local variables)
+[ ] Refactoring (no functional changes, no api changes)
```

Comment:
Are these types of PRs we usually accept? I haven't seen any PRs accepted lately that are purely styling or refactor based. Calling these out might suggest we are open to these types of changes.

Reply:
I know the PR was updated already, but I figured I'll reply for just information sharing: the short answer is no, but the long answer is that PRs are accepted as long as they provide some kind of merit; for example, if there is a demonstrable perf improvement.